.github/workflows/ci.yaml

```diff
@@ -38,9 +38,7 @@ jobs:
 
     steps:
       - name: Checkout Repository
-        uses: actions/checkout@v2
-        with:
-          fetch-depth: 0
```

Comment:
From the actions/checkout docs:

```
Only a single commit is fetched by default, for the ref/SHA that triggered the workflow. Set fetch-depth: 0 to fetch all history for all branches and tags.
```

To deploy the NeXus docs we run the CI with deploy=1 for a specific tag, not the main branch right?

But I guess in that case the "single commit" mentioned corresponds to the tag anyway.

Reply:
Yes, it should be the tip of the current branch.